pkgpanda/cli.py

```diff
@@ -141,6 +141,9 @@ def main():
         except ValidationError as ex:
             print("Validation Error: {0}".format(ex))
             sys.exit(1)
+        except Exception as ex:
```

Comment:
All of these have the same exception handling for ValidationError, PackageError, and Exception. (Exception adds `ERROR:` to the start of the line). Should move the exception handling to surround the argument dispatching code rather than duplicate it in each dispatch